api/controllers/error_test.go

```diff
@@ -0,0 +1,91 @@
+package controllers
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/quintilesims/layer0/common/errors"
+	"github.com/quintilesims/layer0/common/models"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestErrorHandler(t *testing.T) {
+	cases := []struct {
```

Comment:
[nit] I think this reads better and is inline with our style. 

`
	type ErrorCases struct {
		Name           string
		Error          error
		ExpectedStatus int
		ExpectedCode   errors.ErrorCode
	}

	cases := []ErrorCases{
`

Reply:
anonymous structs are pretty common for table tests in go: https://dave.cheney.net/2013/06/09/writing-table-driven-tests-in-go